app/assets/javascripts/components/authentication/reset_password_modal.es6.jsx

```diff
@@ -51,6 +53,17 @@ class ResetPasswordModal extends React.Component {
     }
   }
 
+  renderErrors() {
```

Comment:
same with here, but if `response.message` is an array then maybe you just wanna display the first error